openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/ActivityFeedCard.tsx

```diff
@@ -121,9 +121,13 @@ const ActivityFeedCard: FC<ActivityFeedCardProp> = ({
 
   return (
     <div
-      className={classNames(className, 'hover:tw-bg-gray-100', {
-        'tw-bg-gray-100': visible,
-      })}>
+      className={classNames(
+        className,
+        'hover:tw-bg-gray-100 tw--mx-2.5 tw-px-2.5 tw--mt-1 tw-py-1 tw-mb-1 tw-rounded-4px',
```

Comment:
@ShaileshParmar11 , We have a `tw-rounded` class that will apply border radius `4px`.

```suggestion
        'hover:tw-bg-gray-100 tw--mx-2.5 tw-px-2.5 tw--mt-1 tw-py-1 tw-mb-1 tw-rounded',
```

Reply:
Ohh, totally forgot about this Thanks 🙏🏻